universe/Universe.cpp

```diff
@@ -1291,6 +1291,17 @@ void Universe::ApplyAppearanceEffects() {
     ExecuteEffects(targets_causes, false, false, true);
 }
 
+void Universe::ApplyGenerateSitRepEffects() {
+    ScopedTimer timer("Universe::ApplyGenerateSitRepEffects on all objects");
+
+    // cache all activation and scoping condition results before applying
```

Comment:
I had considered changing that line (I had simply copied it along with the rest of ApplyAppearanceEffects, where it has the same not-quite-full appicability), but the actual caching is still necessarily done as part of the process, isn't it?  It's just that for the ApplyAppearanceEffects and ApplyGenerateSitRepEffects it's really done simply to share code with other calls where that purpose does apply.  So I just left the comment as-is rather than tailoring it more.  

To tailor it more, I suppose the portion ```...since the application of these Effects may affect...``` could be changed to ```...since the application of Effects in general (even if not these particular Effects) may affect...``` both here and in ApplyAppearanceEffects.  Would you prefer that?